### ISSUE TITLE
warning

### ISSUE THREAD

**Issue by KelvinHuang666** [zylo117/Yet-Another-EfficientDet-Pytorch#99]
[Warning] Ignoring Error(s) in loading state_dict for EfficientDetBackbone:
        size mismatch for classifier.header.pointwise_conv.conv.weight: copying a param with shape torch.Size([810, 112, 1, 1]) from checkpoint, the shape in current model is torch.Size([9, 112, 1, 1]).
        size mismatch for classifier.header.pointwise_conv.conv.bias: copying a param with shape torch.Size([810]) from checkpoint, the shape in current model is torch.Size([9]).
[Warning] Don't panic if you see this, this might be because you load a pretrained weights with different number of classes. The rest of the weights should be loaded already.

**Comment by zylo117** [comment]
Don't panic if you see this, this might be because you load a pretrained weights with different number of classes. The rest of the weights should be loaded already.

**Comment by KelvinHuang666** [comment]
if i train myown dataset(2500training picture and 500 val picture) with one class .how can i set my params(eg:num_epochs)  thanks so much

**Comment by heitorrapela** [comment]
@KelvinHuang666 You need to search for the best config. of params for your dataset. For the number of epochs I suggest that you put a high number of epochs and keep tracking of the valid loss (add some early stopping to it) or save just the best model during the validation.
